test/unit/examples/contextRequests/updateContextTimestamp.json

```diff
@@ -13,7 +13,7 @@
             {
               "name": "TimeInstant",
               "type": "ISO8601",
-              "value": "2015-08-05T07:35:01.468Z"
+              "value": "2015-08-05T07:35:01.468+00:00"
```

Comment:
I guess this is the case of a user that is not using timezone field at provision time (so it default to UTC).

Would it be possible to keep the current syntax for these cases? I mean, "2015-08-05T07:35:01.468Z" instead of "2015-08-05T07:35:01.468+00:00". I know that from a semantic point of view both are equivalent, but let's try to minimize the syntax change for current users not using timezone at all.